src/views/Home.jsx

```diff
@@ -1,8 +1,38 @@
 import './Home.css';
 import { SingleList } from '../components';
+import { useState } from 'react';
+import { createList, useAuth } from '../api';
+import { useNavigate } from 'react-router-dom';
 
 export function Home({ data, setListPath }) {
 	const hasList = data.length !== 0;
+
+	const { user } = useAuth();
+	const userId = user?.uid;
+	const userEmail = user?.email;
+
+	const [inputValue, setInputValue] = useState('');
```

Comment:
Something that I like to see when building React apps is leaning into the idea of composing components. With the introduction of this feature, `Home` now has two very distinct roles: 
- display the lists that the user can access
- create new lists

While this is perfectly functional and a great way to "get something working", when implementing production-ready React you'll often pair changes like these with unit tests written with libraries like [jest](https://jestjs.io/docs/tutorial-react) and [react-testing-library](https://testing-library.com/docs/react-testing-library/intro/). Breaking down components to "own" these individual roles can be an excellent way to easily achieve either good test coverage, or at least a stronger insight into what does and doesn't have test coverage. If a component has 1 role, it can be easy to spot at a glance what the test coverage looks like for that role. When a component starts wearing more hats, things can start to slip through the cracks ☔ 

While writing unit tests isn't in the scope of this project, learning how to write code in ways that make unit testing easier is a skillset that's really good to start exercising; the senior devs at your next role will praise you for it 😄 As someone who spends a ton of my time reviewing code, having elements of code broken down into these "roles" that can then be composed together into a parent component like `Home` lets the me (the reviewer, any reviewer, and especially post-review "I've got to figure out what this code is doing so I can make changes to it" teammates) quickly grasp what is and isn't happening in a specific diff, and can go a long way towards the health of a codebase 🚀

Reply:
We can absolutely cover testing if it's something the group is interested in, we should chat at our next meeting!